spec/active_force/sobject/includes_spec.rb

```diff
@@ -397,9 +397,41 @@ module ActiveForce
         end
       end
 
+      context 'when the associations have scopes' do
+        it 'generates the correct SOQL query' do
+          soql = Blog.includes(active_posts: :impossible_comments).where(id: '123').to_s
+          expect(soql).to eq <<-SOQL.squish
+            SELECT Id, Name, Link__c,
+              (SELECT Id, Title__c, BlogId, IsActive,
+                (SELECT Id, PostId, PosterId__c, FancyPostId, Body__c
+                  FROM Comments__r WHERE (1 = 0))
```

Comment:
`WHERE (1 = 0))` nice.

Reply:
😄 it was one of the pre-defined scopes from the original authors